docs/source/privacy.rst

```diff
@@ -0,0 +1,87 @@
+===============
+Privacy Policy
+===============
+
+:Date: 2023-12-17
+
+Introduction
+============
+
+This Privacy Policy describes how we collect, use, and handle your personal information when you use @Spotted_DMI_bot.
+
+Information We Collect
+======================
+
+Each user is assigned a unique **UserID** by telegram.
+This information is easily obtained by any client or bot and is not to be considered sensitive information.
+In fact, it is necessary for the bot to send the user messages.
+
+How We Use Your Information
+===========================
+
+When a spot or report is submitted, the **UserID** is stored in the database.
```

Comment:
It Is better to use the same name of the telegram API, idk if they use chatid formarly or user id, moreover a good reference link would be better for who wants to get the official information about it from the Telegram API